dnf/cli/progress.py

```diff
@@ -120,8 +121,7 @@ def _update(self, now):
             bar = '='*n + '-'*p
             msg = '%3d%% [%-*s]%s' % (pct, bl, bar, msg)
             left -= bl + 7
-        self.fo.write('%-*.*s%s' % (left, left, text, msg))
-        self.fo.flush()
+        self.message('%-*.*s%s' % (left, left, text, msg))
```

Comment:
There is no need to call `self.message`, you can call `dnf.util._terminal_messenger` directly.